src/settings.rs

```diff
@@ -0,0 +1,27 @@
+#[derive(Debug)]
+pub struct Settings {
+    pub venv_from_stdlib: bool,
+    pub venv_outside_project: bool,
+}
+
+impl Default for Settings {
+    fn default() -> Settings {
+        Settings {
+            venv_from_stdlib: true,
+            venv_outside_project: false,
+        }
+    }
+}
+
+impl Settings {
+    pub fn from_env() -> Settings {
+        let mut res = Settings::default();
+        if std::env::var("DMENV_NO_VENV_STDLIB").is_ok() {
+            res.venv_from_stdlib = false;
+        }
+        if std::env::var("DMENV_VENV_OUTSIDE_PROJECT").is_ok() {
+            res.venv_outside_project = true;
```

Comment:
Wild tangent, probably don't want to take this too seriously, but in a twisted kind of way this looks actually slightly safer and makes sense to me:
```
res.venv_from_stdlib ^= std::env::var("DMENV_NO_VENV_STDLIB").is_ok();
res.venv_outside_project ^= std::env::var("DMENV_VENV_OUTSIDE_PROJECT").is_ok();
```

Because if you write it this way, you can't accidentally have a mismatch where the default value is `true` and the env var also sets it to `true` by mistake. It expressed "env vars toggle my value", which is a bit more DRY.

That said, this is probably horribly unreadable =]

Reply:
yeah it is

Reply:
Thanks, I try.